data/links_p_nom.csv

```diff
@@ -1,20 +1,18 @@
 Gotland 2,Sweden - Västervik 57°43′41″N 16°38′51″E﻿ / ﻿57.72806°N 16.64750°E,Sweden - Yigne 57°35′13″N 18°11′44″E﻿ / ﻿57.58694°N 18.19556°E,99.5(92.9/6.6),150,130.0,1983,Thyr,Supplier: ABB,,16.6475,57.72805555555556,18.195555555555554,57.58694444444444
 Interconnexion France Angleterre (new),France - Les Mandarins 50°54′11″N 1°47′5″E﻿ / ﻿50.90306°N 1.78472°E,UK - Sellindge 51°6′21″N 0°58′32″E﻿ / ﻿51.10583°N 0.97556°E,72(72/0),270,2000.0,1986,Thyr,2 bipolar systems Supplier: Alstom,[1][37],1.7847222222222223,50.903055555555554,0.9755555555555555,51.10583333333334
 Gotland 3,Sweden - Västervik 57°43′41″N 16°38′51″E﻿ / ﻿57.72806°N 16.64750°E,Sweden - Yigne 57°35′13″N 18°11′44″E﻿ / ﻿57.58694°N 18.19556°E,98(98/0),150,130.0,1987,Thyr,Supplier: ABB,,16.6475,57.72805555555556,18.195555555555554,57.58694444444444
-Konti-Skan 2,"Denmark - Vester- Hassing 57°3′46″N 10°5′24″E﻿ / ﻿57.06278°N 10.09000°E",Sweden - Lindome 57°36′24″N 12°6′40″E﻿ / ﻿57.60667°N 12.11111°E,147(87/60),285,300.0,1988,Thyr,Supplier: ABB,,10.09,57.062777777777775,12.11111111111111,57.60666666666667
+Konti-Skan 2,"Denmark - Vester, Hassing 57°3′46″N 10°5′24″E﻿ / ﻿57.06278°N 10.09000°E",Sweden - Lindome 57°36′24″N 12°6′40″E﻿ / ﻿57.60667°N 12.11111°E,147(87/60),285,300.0,1988,Thyr,Supplier: ABB,,10.09,57.062777777777775,12.11111111111111,57.60666666666667
 Fenno-Skan,Finland - Rauma 61°09′07″N 21°37′32″E﻿ / ﻿61.15194°N 21.62556°E,Sweden - Dannebo 60°24′14″N 18°08′10″E﻿ / ﻿60.40389°N 18.13611°E,233(200/33),400,500.0,1989,Thyr,Supplier: ABB,,21.625555555555554,61.151944444444446,18.136111111111113,60.403888888888886
-SACOI 2,Italy - Suvereto 43°3′10″N 10°41′42″E﻿ / ﻿43.05278°N 10.69500°E ( before 1992: Italy - San Dalmazio 43°15′43″N 10°55′05″E﻿ / ﻿43.26194°N 10.91806°E),"France- Lucciana 42°31′40″N 9°26′59″E﻿ / ﻿42.52778°N 9.44972°E",422(118/304),200,300.0,1992,Thyr,Multiterminal scheme Supplier: Alstom,,10.695,43.05277777777778,9.449722222222222,42.52777777777778
-SACOI 2,"France- Lucciana 42°31′40″N 9°26′59″E﻿ / ﻿42.52778°N 9.44972°E"," Codrongianos- Italy 40°39′7″N 8°42′48″E﻿ / ﻿40.65194°N 8.71333°E",422(118/304),200,300.0,1992,Thyr,Multiterminal scheme Supplier: Alstom,,9.449722222222222,42.52777777777778,8.679351,40.65765
+SACOI 2,Italy - Suvereto 43°3′10″N 10°41′42″E﻿ / ﻿43.05278°N 10.69500°E ( before 1992: Italy - San Dalmazio 43°15′43″N 10°55′05″E﻿ / ﻿43.26194°N 10.91806°E),"France- Lucciana 42°31′40″N 9°26′59″E﻿ / ﻿42.52778°N 9.44972°E ; Codrongianos, Italy 40°39′7″N 8°42′48″E﻿ / ﻿40.65194°N 8.71333°E",422(118/304),200,300.0,1992,Thyr,Multiterminal scheme Supplier: Alstom,,10.695,43.05277777777778,9.449722222222222,42.52777777777778
 Skagerrak 3,Denmark - Tjele 56°28′44″N 9°34′1″E﻿ / ﻿56.47889°N 9.56694°E,Norway - Kristiansand 58°15′36″N 7°53′55″E﻿ / ﻿58.26000°N 7.89861°E,230(130/100),350,440.0,1993,Thyr,Supplier: Nexans ABB,[34],9.566944444444445,56.47888888888889,7.898611111111111,58.26
```

Comment:
Were all of these intentional changes?

Reply:
More or less, these came when rerunning the `prepare_links_p_nom` script, this the updated version of the csv file. But perhaps it is better to have only code changes for this PR and update the data later

Reply:
yes, should be part of a separate PR.

Reply:
is reverted now